docs/adr/0061-cozy-integration-set-up.md

```diff
@@ -0,0 +1,71 @@
+# 61. Cozy integration set up
+
+Date: 2025-04-21
+
+## Status
+
+Accepted
+
+## Context
+
+- How to run Cozy locally?
+
+## Steps
+### Install cozy-stack CLI
+
+1. Install CouchDB
+```bash
+docker run -d \
+    --name cozy-stack-couch \
+    -p 5984:5984 \
+    -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=password \
+    -v $HOME/.cozy-stack-couch:/opt/couchdb/data \
+    couchdb:3.3
+curl -X PUT http://admin:password@127.0.0.1:5984/{_users,_replicator}
+```
+2. Install cozy-stack CLI
+- Clone cozy-stack
+```bash
+git clone git@github.com:cozy/cozy-stack.git
+cd cozy-stack
+make
+```
+- Install Go and add to path
+- Create cozy configuration file
+```bash
+cp cozy.example.yaml $HOME/.cozy/cozy.yml
```

Comment:
Should create folder `.cozy` before run this command. Avoid error `No such file or directory
`

```bash
mkdir -p ~/.cozy
```